src/test/java/school/hei/haapi/integration/FeeIT.java

```diff
@@ -137,7 +139,7 @@ void manager_read_ok() throws ApiException {
     List<Fee> actualFees2 = api.getFees(String.valueOf(Fee.StatusEnum.PAID), 1, 10);
```

Comment:
Didn't you change manager_create_ok ? Or if it does not exist then do it ?

And how about the PaymentIT ? When you create some payments, what happened ?